doc/classes/Skeleton3D.xml

```diff
@@ -31,6 +31,17 @@
 				[i]Deprecated soon.[/i]
 			</description>
 		</method>
+		<method name="bone_transform_to_world_transform">
+			<return type="Transform">
+			</return>
+			<argument index="0" name="bone_transform" type="Transform">
+			</argument>
+			<description>
+				Takes the given bone pose/transform and converts it to a world transform, relative to the [Skeleton3D] node.
+				
```

Comment:
Empty line here.

Reply:
Sorry about that! I thought I got them all. It should be fixed now.